### PR TITLE
DRIVERS-2789 Convert Another SDAM Spec File to Markdown

### PR DESCRIPTION
**Pull request by blink1073**
Please complete the following before merging:

- [x] Update changelog.
- [ ] Make sure there are generated JSON files from the YAML test files.
- [ ] Test changes in at least one language driver.
- [ ] Test these changes against all server versions and topologies (including standalone, replica set, sharded
  clusters, and serverless).

### CODE REVIEW

**Comment by blink1073** [comment]
@W-A-James we missed a file last time.  :smile:

File: source/server-discovery-and-monitoring/server-discovery-and-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,1990 @@
...
+Although servers are checked in parallel, the function that actually creates the new TopologyDescription should be
+synchronized so only one thread can run it at a time.
+
+##### Replacing the TopologyDescription
+
+Drivers may use the following pseudocode to guide their implementation. The client object has a lock and a condition
+variable. It uses the lock to ensure that only one new ServerDescription is processed at a time, and it must be acquired
+before invoking this function. Once the client has taken the lock it must do no I/O:
+
+```
```
Comment (W-A-James) [inline source/server-discovery-and-monitoring/server-discovery-and-monitoring.md:1300]: Can we make this a python code block?

**Review by W-A-James — CHANGES_REQUESTED** [review]
Just one small change

File: source/server-discovery-and-monitoring/server-discovery-and-monitoring.md (hunk 1 above)
Comment (blink1073) [reply source/server-discovery-and-monitoring/server-discovery-and-monitoring.md:1300]: Done!

**Pull request merged**